src/main/webapp/definitions/alfresco-core.yaml

```diff
@@ -8564,6 +8564,13 @@ definitions:
         type: string
       title:
         type: string
+      client:
+        description: |
+          Optionally provided in the request query to customise the email sent.
```

Comment:
I would change this to something like:
Optional client name used when sending an email to the end user, defaults to "share" if not provided.

Presuming "share" is the default ;-) Replace as necessary.